### DISCUSSION TITLE
Fix for taskbar crashing when right-clicking on it

### DISCUSSION THREAD

**Discussion by ShadowElixir** [valinet/ExplorerPatcher#1978] [General]
Just go to ExplorerPatcher Properties > Advanced > Do not change the taskbar context menu.
Enable that and restart explorer.
(or you could just enable win10 taskbar)
Edit: For those who cannot find settings, go to the search bar and search for properties, you should find explorerpatcher settings.

**Comment by TLCPlusPlus** [comment]
Yep, I had to uninstall ExplorerPatcher because of this issue.

**Reply by lhdigo** [reply]
Me too...

**Comment by Repairman227** [comment]
I can't right-click my taskbar anymore.  
So I am unable to change settings.
I am also unable to uninstall ExplorerPatcher.

**Reply by ShadowElixir** [reply]
Go to the search bar and search for properties, you should find explorerpatcher settings.

**Reply by pyrates999** [reply]
to run the explorer patcher properties application, run this from the run menu:
C:\Windows\System32\rundll32.exe "C:\Program Files\ExplorerPatcher\ExplorerPatcher.amd64.dll",ZZGUI

**Comment by ShadowElixir** [comment]
This is fixed on the latest update
